web/css/source/expand/variables/_structure.less

```diff
@@ -0,0 +1,4 @@
+//
+//  Components
+//  _____________________________________________
+@page-content__margin-horizontal: @content__indent;
```

Comment:
By the coding standards:
"First and second level comments must be surrounded by empty lines."
Would be good to have empty line after comment.